doc/sal_constraints_and_recommendations.rst

```diff
@@ -381,23 +393,31 @@ If a time-of-data is to be associated with an item it should be named
     * *timestamp[n]* - for an array of times associated with the array item(s) in a topic
     * *timestampName1[n], timestampName2[m]* - for multiple arrays of different times for different array sizes
 
-The time(s) should be obtained using the SAL getCurrentTime() method, which returns a double precision value of TAI time with a resolution of at least 0.001 seconds.
+The time(s) should be obtained using the SAL getCurrentTime() method, which returns a double precision value of TAI time in unix seconds with a resolution of at least 0.001 seconds.
+
+.. _sal_constraints_and_recommendations:ignored-attributes-in-topics:
 
 Ignored Attributes in Topics
 ============================
 
+*Deprecated in SAL 4.1*
```

Comment:
Thank you. Please consider appending a brief explanation of why it is deprecated, e.g. "because topics with no fields are allowed"